### PR TITLE
Move to clang-format-11 and start format checks for pdns/modules

### PR DESCRIPTION
**Pull request by omoerbeek**
### Short description

So I think we agree we should enforce formatting in more parts. So start with modules.

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

**Comment by Habbie** [comment]
```
./build-scripts/format-code pdns/recursordist/*.[ch][ch] $(find pdns/modules -name '*.[ch][ch]'
git --no-pager diff
exit $(git diff | wc -l)

/bin/bash: -c: line 0: unexpected EOF while looking for matching `)'
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -546,7 +546,7 @@ jobs:
       - run:
           name: Check formatting
           command: |
-            ./build-scripts/format-code pdns/recursordist/*.[ch][ch]
+            ./build-scripts/format-code pdns/recursordist/*.[ch][ch] $(find pdns/modules -name '*.[ch][ch]'
```
Comment (Habbie) [inline .circleci/config.yml:549]:
```suggestion
            ./build-scripts/format-code pdns/recursordist/*.[ch][ch] $(find pdns/modules -name '*.[ch][ch]')
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -546,7 +546,7 @@ jobs:
       - run:
           name: Check formatting
           command: |
-            ./build-scripts/format-code pdns/recursordist/*.[ch][ch]
+            ./build-scripts/format-code pdns/recursordist/*.[ch][ch] $(find pdns/modules -name '*.[ch][ch]')
```
Comment (Habbie) [inline .circleci/config.yml:549]:
```suggestion
            ./build-scripts/format-code pdns/recursordist/*.[ch][ch] $(find modules -name '*.[ch][ch]')
```

**Comment by Habbie** [comment]
formatting failed, quietly, for two reasons:

(a) wrong path (see suggested fix above)

(b):
```
YAML:32:20: error: unknown key 'SpaceInEmptyBlock'
SpaceInEmptyBlock: false
                   ^~~~~
Error reading /opt/project/.clang-format: Invalid argument
```

**Comment by Habbie** [comment]
All changes in `modules/` are from clang-format, no other manual changes?

**Comment by omoerbeek** [comment]
All changes were automatically generated locally with `clang-format-10` and then verified by `clang-format-11` on CI.

**Comment by Habbie** [comment]
> All changes were automatically generated locally with `clang-format-10` and then verified by `clang-format-11` on CI.

Wonderful, especially the version mix :)

**Review by Habbie — APPROVED** [review]

**Pull request merged**